wazo_sysconfd/exceptions.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from http.server import BaseHTTPRequestHandler
+
+
+class HttpReqError(Exception):
+    def __init__(self, code: int, message: str = None):
+        self.code = code
+        self.message = message or BaseHTTPRequestHandler.responses[code][1]
```

Comment:
The dependency on `BaseHTTPRequestHandler` looks brittle to me: what if `code` is not a standard code? Furthermore, returning a standard message is not very useful.

Reply:
BaseHTTPRequestHandler removed as suggested. thx.